### PR TITLE
Scripts are not working

### PR DESCRIPTION
**Pull request by chillyjim**
Pulling the scripts to try and find the error

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@chillyjim__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@chillyjim__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: checkpoint-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Deploy a Check Point Cluster with multiple Network Interfaces
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fcheckpoint-cluster%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
```
Comment (singhkay) [inline checkpoint-cluster/README.md]: Add the visualize button as well

File: checkpoint-cluster/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,802 @@
...
+        "password",
+        "sshPublicKey"
+      ],
+      "defaultValue": "password",
+      "metadata": {
+        "description": "Authentication type"
+      },
+      "type": "string"
+    },
+    "newStorageAccountName": {
```
Comment (singhkay) [inline checkpoint-cluster/azuredeploy.json]: Eliminate this, see #944 for a better way

File: checkpoint-cluster/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a Check Point Cluster",
+  "description": "This template creates a VNET with multiple subnets and a Check Point Cluster.",
+  "summary": "This template creates a VNET with multiple subnets and a Check Point Cluster",
+  "githubUsername": "chkp-zegman",
+  "dateUpdated": "2016-01-27"
```
Comment (singhkay) [inline checkpoint-cluster/metadata.json]: Don't forget to update this date on the final commit

**Comment by singhkay** [comment]
@chillyjim The build is failing. Can you check the error message? https://travis-ci.org/Azure/azure-quickstart-templates/builds/112243402

**Comment by singhkay** [comment]
@chillyjim Can you resolve the merge conflict?

**Comment by singhkay** [comment]
@chillyjim Are you still working on this?

**Comment by bmoore-msft** [comment]
Closing due to no response

**Pull request closed without merging**